MoviesApp/MoviesApp/Services/ImageService.swift

```diff
@@ -0,0 +1,39 @@
+// ImageService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Сервис кеширования изображений
+struct ImageService: ImageServicable {
+    // MARK: - Private properties
+
+    private let networkService = NetworkService()
+
+    // MARK: - Public properties
+
+    var cache = NSCache<NSString, UIImage>()
```

Comment:
```suggestion
    var cache = NSCache<NSString, UIImage>()
```
UIImage лучше тут Data что б не использовать в сервисе UIKit